strapiExperiment/.envExample

```diff
@@ -0,0 +1,8 @@
+DATABASE_CLIENT=mongo  
```

Comment:
theres a convention for examples files to be named `.env.example` . There are actualy vscode extensions that plug into those file extensions

Reply:
the snake caps was a typo on my part, though the convention I've always followed was .env-example.  Didn't know about the vscode extension

Reply:
all gucci. I'm not caring of either